### ISSUE TITLE
Multi selection permissions issue

### ISSUE THREAD

**Issue by thwaller** [mate-desktop/caja#1019]
#### Expected behaviour
When I select multiple files, click properties and change the file permissions, I would expect that all selected files would change to what I have selected.

#### Actual behaviour
It does not actually change any file permissions when multiple files are selected.

#### Steps to reproduce the behaviour
select multiple files, click properties and change the file permissions, Click the close button. There is no other button to click like save or apply.

#### MATE general version
1.20.1

#### Package version
1.20.2-4ubuntu1

#### Linux Distribution
Ubuntu 18.04

#### Link to downstream report of your Distribution
?

NOTES: When I do not select multiple files (select only one file) the steps above work properly and as expected.

**Comment by vkareh** [comment]
The first time I tried this, I thought I was able to reproduce, but every other time it seemed to work. Upon closer inspection, it seems as though there is an intermediate step in between an enabled and a disabled permission. A screenshot might help explain:

![screenshot at 2018-06-17 22-35-41](https://user-images.githubusercontent.com/259780/41515787-03acfc86-727f-11e8-9cf9-a3db0e9ce375.png)

Notice where the pointer is, in the _Others: Write_ permission. This group of files had it disabled, when I clicked the first time I got that symbol (not a checkmark, just a line). When clicking on it a second time, I got the proper checkbox, the _Text view_ at the bottom changed to `-rw-rw-rw-`, and the terminal showed the correct new permission.

So this works, but that intermediate step makes it really confusing. @thwaller - does this represent what happened on your side? Would you be able to verify whether this is what happened to you?

**Comment by thwaller** [comment]
My initial response would have been no, that is not what happened to me. But after looking in detail and using your reply as a guide, I found that it is in fact the same issue. It only happens like this on multiple selections, or a folder selection. Single file selections are empty box or check mark only.

I mentioned my initial response because I did not even notice this at first. It is really hard to notice it if not paying close attention. You expect a check or no check behaviour, and by having the dash to check change while keeping the color change to green the same makes it almost unnoticeable. Would have been nice to change it to a yellow then a green or something....

Anyway, I have no idea what the dash even means. There is nothing in the help about it either. The behaviour I would expect is that I check the box to allow the appropriate permission and uncheck it to disallow it. If I am selecting multiple items, I would expect the thought to be that I am intending to do something to the whole of my selection. Apparently that is no longer the proper assumption to make with Caja? I checked on Nautilus and its behaviour is what I would expect, or what I assume to be normal behaviour.

**Comment by vkareh** [comment]
I agree, that green box is baffling, and we need to either show a message explaining it (assuming it serves a propose) or remove that glitch

**Comment by rcaridade145** [comment]
@vkareh @thwaller  Found this https://askubuntu.com/questions/143477/what-do-dashes-mean-in-permissions-tab-of-file-folder-properties .   Basically it states that there are files with different permissions selected for that user/action.
